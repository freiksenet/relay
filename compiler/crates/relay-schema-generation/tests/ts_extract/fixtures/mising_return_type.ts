/**
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

//extract
function id(user: User) {}

//extract
function number(user: User) {}

//extract
function string(user: User) {}

