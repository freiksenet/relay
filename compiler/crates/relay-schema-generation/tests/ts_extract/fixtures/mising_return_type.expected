==================================== INPUT ====================================
/**
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

//extract
function id(user: User) {}

//extract
function number(user: User) {}

//extract
function string(user: User) {}
==================================== OUTPUT ===================================
✖︎ Expected the function return type to exist

  <generated>:9:1
    7 │ 
    8 │ //extract
    9 │ function id(user: User) {}
      │ ^^^^^^^^^^^^^^^^^^^^^^^^^^
   10 │ 


✖︎ Expected the function return type to exist

  <generated>:12:1
   10 │ 
   11 │ //extract
   12 │ function number(user: User) {}
      │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   13 │ 


✖︎ Expected the function return type to exist

  <generated>:15:1
   13 │ 
   14 │ //extract
   15 │ function string(user: User) {}
      │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   16 │
